### ISSUE TITLE
A-10C_2 Lighting Panel Axis

### ISSUE THREAD

**Issue by big32dave** [Munkwolf/dcs-community-keybinds#54]
When I assign an axis to one of the lighting panel knobs (i.e. Formation Lights), it is only reading half of the axis.  The knob in the cockpit doesn't move until the physical knob on my controller is between 50-100%

**Comment by Munkwolf** [comment]
Yeah, unfortunately depending on how the module is built some of the axis can behave that way since some commands weren't built with the intention of them being available as an axis. There's no way to control that via the keybinds configuration unless there is a different command available with the full range.

What I do to work around it is use axis tune so the range is from 50-100 instead of 0-100. Then the full axis will work as expected.

**Comment by danw-eth** [comment]
This is an example of the axis tune, I use for Harrier HUD Brightness axis:
https://imgur.com/gallery/Bv8KH23
